gym-unity/gym_unity/envs/__init__.py

```diff
@@ -51,6 +51,10 @@ def __init__(
         self._env = UnityEnvironment(
             environment_filename, worker_id, no_graphics=no_graphics
         )
+
+        # Take a single step so that the brain information will be sent over
+        self._env.step()
```

Comment:
@vincentpierre Any other suggestions on how to do this?

Reply:
I do not know of a good way. maybe you could check first if there are no brains at all ?

Reply:
Good call.